lib/reply.js

```diff
@@ -130,6 +130,10 @@ Reply.prototype.send = function (payload) {
     return this
   }
 
+  if (this.request.raw && this.request.raw.aborted === true) {
```

Comment:
Could you add a TODO comment here to use the `destroyed` field (see https://nodejs.org/dist/latest-v16.x/docs/api/http.html#messageaborted, aborted is deprecated) as soon as we drop support for Node v14? Thanks

I also think that `raw` would always be defined here. When is it not?

Reply:
`raw` should be always defined according to the types but in this case a few test cases would be broken. I had to choose to add `raw` to these tests or do this. Would you prefer if I update those test cases?
Will do the rest.